includes/class-sensei-utils.php

```diff
@@ -2489,6 +2491,96 @@ public static function is_woocommerce_active( $minimum_version = null ) {
 		return true;
 	}
 
+	/**
+	 * Get WooCommerce plugin information.
+	 *
+	 * @return array WooCommerce information.
+	 */
+	public static function get_woocommerce_plugin_information() {
```

Comment:
Both methods moved from Setup Wizard class, since this is used for a different context now.